src/main/java/pages/LeftPanelPage.java

```diff
@@ -0,0 +1,36 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+
+public class LeftPanelPage extends BasePage {
+
+    private By projectBox = By.cssSelector(".expansion_panel__header");
+    private By addSymbolButton = By.cssSelector("button[aria-label=\"Add Project\"]");
```

Comment:
use single quotes instead of double quotes in this way you don't need to escape the double quotes